src/modules/movies.js

```diff
@@ -37,7 +41,7 @@ class Movies {
         </div>
       </div>
       <div class="title-container">
-        <h4><b>${movie.name}</b></h4> 
+        <h4><b>${movie.name} ${movieNumber}</b></h4> 
         <div class="like-icon"> <i class="fa fa-heart" data-pos=${movie.id}></i> <span id="movie-id"> ${msgLikes} </span> Like(s)</div>
       </div>
       `;
```

Comment:
hey @betses  👍 
good thinking by doing these changes. It'll make a layout more precise about movie list numbers and names 😄👍